content/blog/simplify-embedded-projects-with-esp-bsp/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Simplify Your Embedded Projects with ESP-BSP"
+date: 2024-06-18
+showAuthor: false
+authors:
+    - "juraj-michalek"
+tags: ["Embedded Systems", "ESP32", "ESP32-S3", "Espressif", "BSP"]
+---
+
+# Simplify Your Embedded Projects with ESP-BSP
+
+## Introduction
+
+Are you a maker or an embedded systems enthusiast looking to create applications that work seamlessly across different ESP32 development boards? Whether you’re using the ESP-Wrover-Kit, M5Stack-CoreS3, ESP32-S3-BOX-3, or other compatible boards, the ESP Board Support Package (ESP-BSP) makes your life easier. In this article, we’ll walk you through how to get started with ESP-BSP, enabling you to focus on your project’s functionality without worrying about hardware differences.
+
+## What is ESP-BSP?
+
+ESP-BSP is a collection of Board Support Packages specifically designed for Espressif’s development boards. It provides a standardized interface, allowing you to develop applications that are easy to port between different boards. By using ESP-BSP, you can:
+
+- **Streamline Hardware Integration**: Simplify code and reduce complexity.
+- **Enhance Portability**: Easily adapt your application to different boards.
+- **Access Standardized APIs**: Ensure consistency across your projects.
+
+## Getting Started with ESP-BSP
+
+### Hardware Setup
+
+Ensure you have the following hardware:
+
+- [ESP32-S3-BOX-3](https://github.com/espressif/esp-box) development board.
+- USB-C Cable for power and programming.
+
+### Prerequisites
+
+Before you begin, make sure you have the following:
+
+- [ESP-IDF](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/get-started/index.html#installation): The official development framework for the ESP32, properly installed and sourced in your shell.
```

Comment:
Since we are using specific version of the package below, I would add a note here, which IDF version was used when writing this tutorial.

Reply:
Yep, make sense. I will fix it to 5.3.